README.md

```diff
@@ -7,65 +7,86 @@ This getting started will show you how to work with MapR-DB JSON using:
 * MapR-DB Shell to interact with JSON tables from the command line
 * Apache Drill to so some analytics with SQL
 * Java and the OJAI library to build operational application.
+* Spark-DB connector to natively load / store RDDs to table.
 
 You will learn how to optimize query using MapR-DB Secondary indexes.
 
 **Prerequisites**
 
-* MapR Converged Data Platform 6.0 Beta with Apache Drill
+* MapR Converged Data Platform 6.0 Beta with Apache Drill OR [MapR Developer Sandbox Container](https://maprdocs.mapr.com/home/MapRContainerDevelopers/MapRContainerDevelopersOverview.html).
 * JDK 8
 * Maven 3.x
 
+## Setting up MapR Developer Sandbox Container
 
+MapR Developer Sandbox Container is an effort to get MapR running on your developer machine so that one could quickly get in to working within MapR environment.
+
+Installation, Setup and further information can be found [**here**](https://maprdocs.mapr.com/home/MapRContainerDevelopers/MapRContainerDevelopersOverview.html).
 
 ## Importing Data into MapR JSON Tables
 
 The application use the Yelp Academy Dataset that contains a list of Business and User Reviews.
 
-#### 1- Download the Yelp Dataset from [here](https://www.yelp.com/dataset_challenge)
+#### 1. Download the Yelp Dataset from [here](https://www.yelp.com/dataset_challenge)
 
-#### 2- Copy the dataset to MapR Cluster
+#### 2. Unarchive the dataset
 
 ```
-scp yelp_dataset_challenge_round9.tar mapr@mapr60:/mapr/my.cluster.com/tmp/
+$ tar xvf yelp_dataset_challenge_round9.tar
 ```
 
-* `mapr60` is one of the node of your cluster
-* `my.cluster.com` is the name of your cluster
+#### 3. Copy the dataset to MapR Cluster
+
+##### 3a. Copy data to the node 
+To do this, we first copy the file (say `business.json`) to the cluster node.
```

Comment:
Also change "file" to "files" since your example shows multiple files being copied.